docs/release-notes/8.0.asciidoc

```diff
@@ -46,23 +51,23 @@ TIP: You can search for the rule by entering the rule name in the Rule table's s
 . Save the changes.
 . Activate the rule.
 
-[discrete]
-[[upgrade-reqs-8.0.0]]
-==== Upgrade requirements
-* Rules must be manually disabled before upgrading to {stack} version 8.0.0, and then manually enabled after the process completes to avoid gaps in rule coverage. Refer to <<upgrade-intro, Upgrade {elastic-sec}>> for more information.
-* To make the *Data view* option appear in an environment with legacy alerts, a user with elevated role privileges must visit the {es-sec-app}, open a page that displays alert data (such as the Overview page), then refresh the page. The user's role privileges must allow them to enable the detections feature in a Kibana space. Refer to <<enable-detections-ui, Enable and access detections>> for more information.
+*The import process fails for rules with exception comments*
 
-+
-NOTE: If new alerts are generated in an upgraded environment without legacy alerts, refreshing any page with alert data in {elastic-sec} will make the *Data view* option appear in the {es-sec-ui}.
+Comments on rule exceptions will cause the import process to fail, as the following system-generated fields cannot be validated for exception comments ({pull}124742[#124742]):
```

Comment:
```suggestion
Comments on rule exceptions cause the import process to fail, because the following system-generated fields cannot be validated for exception comments ({pull}124742[#124742]):
```